command/assets/example.nomad

```diff
@@ -0,0 +1,394 @@
+# There can only be a single job definition per file. This job is named
+# "example" so it will create a job with the ID and Name "example".
```

Comment:
could you run [`hclfmt`](https://github.com/fatih/hclfmt) over these as quick double check for consistent formatting??

Reply:
Just was looking at that and it looks like `hclfmt` is mangling a few of the multi-line comments in the bigger `example.nomad` file. I'm in the middle of trying to figure out why but once I've got it figured out I will do that.

Reply:
I think what I was seeing may be related to https://github.com/hashicorp/hcl/issues/253 but I've been able to fix it by ensuring that the comments following a top-level object are always followed by a non-comment line.

That is:
```diff
@@ -15,7 +15,7 @@ job "example" {
   # The "region" parameter specifies the region in which to execute the job.
   # If omitted, this inherits the default region name of "global".
   # region = "global"
-
+  #
   # The "datacenters" parameter specifies the list of datacenters which should
   # be considered when placing this task. This must be provided.
   datacenters = ["dc1"]

```